server/integration_tests/explore_test.py

```diff
@@ -323,6 +336,11 @@ def test_e2e_superlatives(self):
     ],
                                 test='unittest')
 
+  def test_e2e_translate(self):
+    # Chinese query for "which cities in the Santa Clara County have the highest larceny?"
+    self.run_detect_and_fulfill('e2e_translate_chinese', ['圣克拉拉县哪些城市的盗窃率最高？'],
```

Comment:
To Bo's point, maybe we can also add another query "what about car theft?", exercise context.

Reply:
Good idea both. Done.